### PR TITLE
tweak: Add new paged layout variation for pre-fare left body region

### PR DESCRIPTION
**Pull request by jzimbel-mbta**
**Asana task**: [Update Pre-Fare template](https://app.asana.com/0/1185117109217413/1204649346992049/f)

- [x] Tests added?

### CODE REVIEW

File: assets/package-lock.json
Comment (cmaddox5) [inline assets/package-lock.json:1]: Are these changes from running a fresh `npm install`? Just want to make sure the changes are intentional.

**Review by cmaddox5 — APPROVED** [review]
Looks good! Just one non-blocking question.

File: assets/package-lock.json
Comment (jzimbel-mbta) [reply assets/package-lock.json:1]: Yeah, I needed to run `npm --prefix assets install` to pick up Brett's recent version upgrades. Before that, the page was loading but didn't seem to run any JS, so it remained blank (without any exceptions thrown).

Comment (jzimbel-mbta) [reply assets/package-lock.json:1]: And I believe you're supposed to check in changes to package-lock.json, so I did that 🤷

**Pull request merged**